File: Integration Examples/eAPI v1.8/java/eAPI-v18-connector/src/main/java/cz/monetplus/mips/eapi/v18/connector/entity/PayRes.java
package cz.monetplus.mips.eapi.v18.connector.entity;

import java.util.List;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

import cz.monetplus.mips.eapi.v18.connector.entity.ext.Extension;

@XmlRootElement
@XmlAccessorType(XmlAccessType.FIELD)
public class PayRes extends SignBase {

	private static final long serialVersionUID = -3825192932302805075L;
	
	@XmlElement
	public String payId;

	@XmlElement
	public int resultCode;

	@XmlElement
	public String resultMessage;

	@XmlElement
	public Integer paymentStatus;

	@XmlElement
	public String authCode;

	@XmlElement
	public String customerCode;

	@XmlElement
	public List<Extension> extensions;

	public PayRes() {
		
	}

	@Override
	public String toSign() {
		StringBuilder sb = new StringBuilder(); 
		add(sb, payId);
		add(sb, dttm);
		add(sb, resultCode);
		add(sb, resultMessage);
		add(sb, paymentStatus);
		add(sb, authCode);
		add(sb, customerCode);
		deleteLast(sb);
		return sb.toString();
	}

}

File: examples/eApi v1.9/dotNET/CsobGTWDemo/CsobGatewayClientExample/Communication/DataObjects/Response/PayRes.cs
using System.Text;
using Action = CsobGatewayClientExample.Communication.DataObjects.Act.Action;

namespace CsobGatewayClientExample.Communication.DataObjects.Response;

public class PayRes : SignBase
{
    private string? PayId { get; set; }

    private int ResultCode { get; set; }

    private string? ResultMessage { get; set; }

    private int PaymentStatus { get; set; }

    private string? AuthCode { get; set; }

    public List<string>? Extensions { get; set; }

    private string? StatusDetail { get; set; }

    private Action? Actions { get; set; }

    public override string ToSign()
    {
        var sb = new StringBuilder();
        Add(sb, PayId);
        Add(sb, Dttm);
        Add(sb, ResultCode);
        Add(sb, ResultMessage);
        Add(sb, PaymentStatus);
        Add(sb, AuthCode);
        Add(sb, StatusDetail);
        Add(sb, Actions?.ToSign());
        DeleteLast(sb);
        return sb.ToString();
    }
}